mbuild/tests/test_packing.py

```diff
@@ -305,6 +305,119 @@ def test_packmol_warning(self, h2o):
         with pytest.warns(UserWarning):
             mb.fill_box(h2o, n_compounds=10, box=[1, 1, 1], overlap=10)
 
+    def test_packmol_args(self, h2o):
+        try:
```

Comment:
To make sure the error got raised every time, we should use the `with pytest.raises(Exception)`.